packages/gallery/src/components/gallery/proGallery/galleryContainerNew.js

```diff
@@ -129,7 +129,6 @@ export class GalleryContainer extends React.Component {
     this.scrollToItem(this.props.currentIdx, false, true, 0);
     this.handleNewGalleryStructure();
     this.eventsListener(GALLERY_CONSTS.events.APP_LOADED, {});
-    this.getMoreItemsIfNeeded(0);
```

Comment:
do not remove this from here. It should appear in both cases (shouldReceiveProps doesn't happen on mount)